Algebraic_kernel_d/include/CGAL/Algebraic_kernel_d/Bitstream_descartes_rndl_tree_traits.h

```diff
@@ -28,7 +28,7 @@
 #include <vector>
 
 #if CGAL_USE_CORE
-namespace CORE { class BigInt; }
+#include <CGAL/CORE/BigInt.h>
```

Comment:
That include makes CGAL_Core a new dependency of all CGAL packages.

Why not keep the forward declaration?

Reply:
Because it is only a `typedef` to the boost mp rational type.